t/unit-tests/u-reftable-tree.c

```diff
@@ -64,23 +64,25 @@ static void t_infix_walk(void)
 
 	do {
 		struct tree_node *node = tree_insert(&root, &values[i], t_compare);
-		check(node != NULL);
+		cl_assert(node != NULL);
 		i = (i * 7) % 11;
 		count++;
 	} while (i != 1);
 
 	infix_walk(root, &store, &c);
 	for (i = 1; i < ARRAY_SIZE(values); i++)
-		check_pointer_eq(&values[i], out[i - 1]);
-	check(!out[i - 1]);
-	check_int(c.len, ==, count);
+		cl_assert_equal_p(&values[i], out[i - 1]);
+	cl_assert(out[i - 1] == NULL);
+	cl_assert_equal_i(c.len, count);
 	tree_free(root);
 }
 
-int cmd_main(int argc UNUSED, const char *argv[] UNUSED)
+void test_reftable_tree__tree_search(void)
 {
-	TEST(t_tree_search(), "tree_search works");
-	TEST(t_infix_walk(), "infix_walk works");
+	t_tree_search();
```

Comment:
This is a bit curious. I would've expected you to inline `t_tree_search()` here.